docs/source/development_js.rst

```diff
@@ -1,81 +1,52 @@
 .. _development_js:
 
-Installing Javascript machinery
+Installing JavaScript machinery
 ===============================
 
-Running the Notebook from the source code on GitHub requires some JavaScript
-tools to build/minify the CSS and JavaScript components. We do these steps when
-making releases, so there's no need for these tools when installing released
-versions of the Notebook.
+.. note::
 
-First, install `Node.js <https://nodejs.org/>`_. The installers on the
-Node.js website also include Node's package manager, *npm*. Alternatively,
-install both of these from your package manager. For example, on Ubuntu or Debian::
+    This section is prepared for contributors to the Notebook source code.
+    Users of the released Notebook do not need to install the JavaScript
+    tools.
 
-    sudo apt-get install nodejs-legacy npm
+Building the Notebook from its GitHub source code requires some tools to
+create and minify JavaScript components and the CSS. These tools and the
+following steps are used when making a Notebook release.
 
-You can then build the JavaScript and CSS by running::
+#. Install `Node.js`_ and :program:`npm`.
 
-    python setup.py css js
+   - Using the installers on `Node.js`_ website:
+     Select a pre-built installer
+     on the `Node.js`_ website. The installer will include Node.js and
+     Node's package manager, :program:`npm`.
 
-This will automatically fetch the remaining dependencies (bower, less) and
-install them in a subdirectory.
+   - Using system's package manager:
+     Install Node.js and :program:`npm` using the
+     system's package manager. For example, the command for Ubuntu or Debian
+     is:
 
```

Comment:
You can probably mention conda here as well... because it is trivial to get node with it:

`conda install -c javascript nodejs`

This will also include `npm`...